modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -634,6 +641,174 @@ public void testDeleteCustomObjectEntry() throws Exception {
 		);
 	}
 
+	@Test
+	@TestInfo("LPD-60423")
+	public void testExportAndImportRelatedObjectEntryWithDifferentScope()
+		throws Exception {
+
+		ObjectRelationship objectRelationship1 = _addObjectRelationship(
```

Comment:
As we discussed let’s consolidate into a single test to cover both scenarios.Please move the object relationship creation to the second test, also change its name to `testGetRelatedObjectEntryWithDifferentGroup`. Thanks!